packages/core/test/util/initializedTooLateHeuristic_test.js

```diff
@@ -73,7 +73,7 @@ describe('[UNIT] util.initializedTooLateHeurstic', function () {
       }
 
       require.cache[resolvedPath] = {};
-      const exclude = ['bluebird', 'bunyan', 'winston'];
+      const exclude = ['bluebird', 'bunyan', 'winston', 'request'];
```

Comment:
Q: Why?

Reply:
I invoked `requireHook` on the `onModuleLoad` function for the `request` module to include the deprecated warning. As a result, this module has already been added to the list. Since this list isused in the current test case, it was necessary to exclude the `request` module from the list; otherwise, the test would fail.

Reply:
I mean here https://github.com/instana/nodejs/pull/1004/files#diff-93014d2f82f6d0b77f9edd3762ab8828fa57ff97edf9dd7a0ca114323df081eb

Reply:
Discussed offline: please add the request module to the patterns of the utility 👍